### DISCUSSION TITLE
How to run multiple exes with g-cli simultaneously?

### DISCUSSION THREAD

**Discussion by bienieck** [JamesMc86/G-CLI#143] [Q&A] [answered]
I tried to run multiple exes with g-cli simultaneously. The first app runs fine but when I try to run another one while the first one is still running I'm getting "Error: Failed to create the network listener". Please take a look at the screenshot. What is wrong with my code? Or perhaps the CLI command I used is wrong?

![g-cli-exe-test](https://user-images.githubusercontent.com/48261145/228535690-0764620a-597c-493e-8990-41caa03cb9c1.png)

**Comment by JamesWiresmith** [comment]
This could be related to #144 

There isn't anything special required to support this - what you have done should work.

**Comment by JamesWiresmith** [comment, marked as answer]
Just pushed v3.0.1 to VIPM which should fix this. You can get it now on the release pages if you cant wait!
